src/pipx/commands/inject.py

```diff
@@ -116,3 +132,17 @@ def inject(
 
     # Any failure to install will raise PipxError, otherwise success
     return EXIT_CODE_OK if all_success else EXIT_CODE_INJECT_ERROR
+
+
+def parse_requirements(filename: Union[str, os.PathLike]) -> Generator[str, None, None]:
+    """
+    Extracts package specifications from requirements file.
+
+    Just returns all of the non-empty lines with comments removed.
```

Comment:
This version is PEP 257-compliant ;-)

```suggestion
    Extract package specifications from requirements file.

    Return all of the non-empty lines with comments removed.
```